api/v3rpc/rpctypes/error.go

```diff
@@ -47,6 +47,7 @@ var (
 	ErrGRPCMemberNotLearner       = status.Error(codes.FailedPrecondition, "etcdserver: can only promote a learner member")
 	ErrGRPCLearnerNotReady        = status.Error(codes.FailedPrecondition, "etcdserver: can only promote a learner member which is in sync with leader")
 	ErrGRPCTooManyLearners        = status.Error(codes.FailedPrecondition, "etcdserver: too many learner members in cluster")
+	ErrGPRCIdMismatch             = status.Error(codes.FailedPrecondition, "etcdserver: cluster ID mismatch")
```

Comment:
duplicated definition https://github.com/etcd-io/etcd/blob/693f25d6b98f5070527bc2b461f0992227c01c14/server/etcdserver/api/rafthttp/http.go#L57

Please raise a followup issue to reuse the same error. Also please also search "cluster ID mismatch" in the code base, there are also some hard code error messages. Please also consider to use the error.

Reply:
Note there is also a typo on `ErrGPRCIdMismatch`. 

```suggestion
	ErrGRPCIdMismatch             = status.Error(codes.FailedPrecondition, "etcdserver: cluster ID mismatch")
```

Reply:
Or expose errClusterIDMismatch, and use it directly https://github.com/etcd-io/etcd/blob/693f25d6b98f5070527bc2b461f0992227c01c14/server/etcdserver/api/rafthttp/http.go#L57